src/domain/networking/packets/EntityQuery.ts

```diff
@@ -0,0 +1,116 @@
+//
+//  EntityQuery.ts
+//
+//  Created by Julien Merzoug on 27 Apr 2022.
+//  Copyright 2022 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+import { OctreeQueryFlags } from "../../octree/OctreeQuery";
+import assert from "../../shared/assert";
+import ConicalViewFrustum from "../../shared/ConicalViewFrustum";
+import GLMHelpers from "../../shared/GLMHelpers";
+import PacketTypeValue from "../udt/PacketHeaders";
+import UDT from "../udt/UDT";
+import NLPacket from "../NLPacket";
+
+
+type EntityQueryDetails = {
+    connectionID: number,
+    numFrustums: number,

```

Comment:
This property isn't necessary: the number is the same as the length of the array of frustums.